examples/discovery-search-app/package.json

```diff
@@ -9,9 +9,10 @@
   "scripts": {
     "build": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts build",
     "eject": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts eject",
-    "start": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts start",
+    "start:client": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts start",
+    "start": "concurrently -n 'SERVER,CLIENT' -c 'blue,green' \"yarn run server:run\" \"yarn run start:client\" -k",
```

Comment:
maybe we should name this `start:prod` and have it only run `yarn run server:run`? i'm not sure what the true benefit is to start the server during normal operation

Reply:
If the only thing this command does is run `yarn run server:run`, then I don't see why we need it at all.